testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -3505,6 +3511,7 @@ void checkPeriodPredicate(Checker checker) {
     expr("NULL").same();
   }
 
+
```

Comment:
why the extra line here?

Reply:
It seems that I accidentally clicked the wrong button and submitted it. I'm sorry for this and I will solve it.